opentelemetry-otlp/tests/integration_test/src/test_utils.rs

```diff
@@ -37,28 +34,8 @@ pub static METRICS_FILE: &str = "./actual/metrics.json";
 pub static LOGS_FILE: &str = "./actual/logs.json";
 pub static TRACES_FILE: &str = "./actual/traces.json";
 
-static INIT_TRACING: Once = Once::new();
-
-fn init_tracing() {
-    INIT_TRACING.call_once(|| {
-        // Info and above for all, debug for opentelemetry
-        let filter_fmt =
-            EnvFilter::new("info").add_directive("opentelemetry=debug".parse().unwrap());
-        let fmt_layer = tracing_subscriber::fmt::layer()
-            .with_thread_names(true)
-            .with_filter(filter_fmt);
-
-        // Initialize the tracing subscriber with the OpenTelemetry layer and the
-        // Fmt layer.
-        tracing_subscriber::registry().with(fmt_layer).init();
-        otel_info!(name: "tracing::fmt initializing completed! SDK internal logs will be printed to stdout.");
-    });
-}
-
 #[allow(clippy::await_holding_lock)]
 pub async fn start_collector_container() -> Result<()> {
-    init_tracing();
```

Comment:
If I recall correctly, this is not redundant.

Reply:
Hey @lalitb , this is the only usage of that function, and I don't think that it achieves anything apart from getting logs out to the console. Can you elaborate - maybe i'm missing something?

Reply:
No, you are right. Sorry for confusion.

Reply:
No worries!